packages/app-builder/src/routes/_builder+/data.tsx

```diff
@@ -143,6 +164,22 @@ function TableDetails({
             : t('data:required');
         },
       },
+      {
+        id: 'unicityConstraint',
+        accessorKey: 'unicityConstraint',
+        size: 30,
```

Comment:
Not enough space for small screen.

A rule of thumb: 
- this settings is the minimal pixel length required for a column (so see it as a `min-width` value)
- the relative comparison between each column `size` is used to allow space accorss columns (when there is enough space to at least, respect the `min-width` constraint)


So in the end :
- check for small screen that the header is not clipped
- adapt if needed the size of other columns to distribute space accross columns

As a last resort, you can use `minSize` and/or `maxSize`

Reply:
oh shit yes, I was developing on the big screen and completely forgot about that.

Reply:
I'm trying 70 which seems to be about enough when the width is 1350px